### PR TITLE
Added Shimmer component

### PR DESCRIPTION
**Pull request by Rohanhacker**

### CODE REVIEW

**Comment by arorayash** [comment]
Please check this on Safari...seems filter is not working well, the shimmer looks fragmented. 
Screenshot 2019-10-14 at 5 02 39 PM

File: src/components/Shimmer/Shimmer.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import styled, { keyframes } from "styled-components"
+
+type heightType = "small" | "medium" | "large"
+
+export interface IShimmerProps {
```
Comment (prasook-jain) [inline src/components/Shimmer/Shimmer.tsx]: You can `import { BaseComponentProps } from "../../common/types"` for `style` & `className`, so in future anything general like style & className need to be added to all component, can be added to `BaseComponentProps`

File: src/components/Shimmer/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Shimmer
+
+## Usage
+
+```javascript
+import { Shimmer } from "KnitUI"
+
```
Comment (prasook-jain) [inline src/components/Shimmer/README.md]: Please create a table of props ( height, and other optional props that can be passed), also mention values that can be passed in props.

Comment (prasook-jain) [inline src/components/Shimmer/README.md]: Please create a table of props ( height, and other optional props that can be passed), also mention values that can be passed in those props.

Comment (Kushagra8888) [reply src/components/Shimmer/README.md]: rejected

**Pull request merged**